leakcanary-android-core/src/main/java/leakcanary/internal/HeapDumpPolicy.kt

```diff
@@ -0,0 +1,28 @@
+package leakcanary.internal
+
+import leakcanary.LeakCanary
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.DISABLED_CONFIG_CHANGED
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.DISABLED_DEBUGGER_ATTACHED
+import leakcanary.internal.HeapDumpPolicy.HeapDumpStatus.ENABLED
+
+object HeapDumpPolicy {
```

Comment:
Does this need to live in the `leakcanary-android-core` module or could we move it and then make it internal?

Reply:
I moved it inside AboutScreen and made it private instead. Let me know if that's better.